src/main/java/com/jobs/portal/jobsportal/service/impl/JobServiceImpl.java

```diff
@@ -30,4 +36,30 @@
 
         return jobShifts;
     }
+
+    @Override
+    public Integer postJob(JobPostRequest jobPostRequest){
+        JobEntity jobPost = new JobEntity();
+        jobPost.setUsername(jobPostRequest.getUsername());
```

Comment:
Use Builder, Instead on Setter and Getters

Reply:
Done